### PR TITLE
add stringcase 1.2.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

A new dependency of https://github.com/conda-forge/monkeytype-feedstock/pull/11, but have actually wanted in the past anyway.

- tests are not packaged
- upstream does not have tags from which to grab the tests

So import test it is!

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/help-python this appears ready for a look!

File: recipes/python-stringcase/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{% set name = "stringcase" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: python-{{ name }}
```
Comment (beckermr) [inline recipes/python-stringcase/meta.yaml]: We should use the same name as pypi? Or is there a name conflict?

**Comment by bollwyvl** [comment]
The upstream repo is named python-stringcase.... I'm good either way, but
afk.

**Comment by beckermr** [comment]
I don't mean the repo, I mean the pypi package. We should match that I think.

https://pypi.org/project/stringcase/

File: recipes/python-stringcase/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/python-stringcase/meta.yaml]: renamed

**Comment by bollwyvl** [comment]
Thanks!

**Pull request merged**